django/contrib/admin/static/admin/css/nav_sidebar.css

```diff
@@ -118,3 +118,19 @@
         max-width: 100%;
     }
 }
+
+#nav-filter {
+    width: 100%;
+    box-sizing: border-box;
+    padding: 5px 16px;
+    margin: 5px 0;
+    border: none;
+}
+
+#nav-filter.no-results {
+    background: #ffefef;
+}
+
+#nav-sidebar table {
+    width: 100%;
+}
```

Comment:
This is how it looks like in case there are no search results

<img width="314" alt="sidebar" src="https://user-images.githubusercontent.com/1742065/112629448-67c84d80-8e45-11eb-82c4-cda9a15c7771.png">